doc/release-notes.md

```diff
@@ -169,19 +169,22 @@ Relay and Mining: Priority transactions
 ---------------------------------------
 
 Transactions that do not pay the minimum relay fee, are called "free
-transactions" or priority transactions. Previous versions of Bitcoin
-Core would relay and mine priority transactions depending on their
-setting of `-limitfreerelay=<r>` (default: `r=15` kB per minute) and
-`-blockprioritysize=<s>` (default: `50000` bytes of a block's
-priority space).
-
-Priority code is scheduled for removal in Bitcoin Core 0.13. In
-Bitcoin Core 0.12, the default block priority size has been set to `0`
-and the priority calculation has been simplified to only include the
-coin age of inputs that were in the blockchain at the time the transaction
-was accepted into the mempool.  In addition priority transactions are not
-accepted to the mempool if mempool limiting has triggered a higher effective
-minimum relay fee.
+transactions" or priority transactions. Bitcoin Core relays and mines
+priority transactions depending on the setting of `-limitfreerelay=<r>`
+(default: `r=15` kB per minute) and `-blockprioritysize=<s>`.
+
+In Bitcoin Core 0.12, priority transactions are not accepted to the mempool nor
+relayed if mempool limiting has triggered a higher effective minimum relay fee.
+
+Mining of priority transactions is also now disabled by default. To re-enable
```

Comment:
This sounds odd if you don't mention the reason: `Priority code is scheduled for removal in Bitcoin Core 0.13`